helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts

```diff
@@ -145,6 +147,14 @@ export class CheckChatStatus {
 							}
 							await checkAgentStatusDirectCallback.checkAgentStatusCallbackError(this.technicalDifficultyMessage);
 							return;
+						} else if (isAgentDisconnected === true) {
```

Comment:
In this case can you show me where isAgentDisconnected was set?

Reply:
https://github.com/WideChat/Apps.Salesforce.LiveAgent/blob/d1615f7794729fd49305538b54078b8bf160df9d/helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts#L123